friendlyarm/nanopc-t4/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, pkgs, ... }:
+
+{
+  boot.loader = {
+    grub.enable = lib.mkDefault false;
+    # Enables the generation of /boot/extlinux/extlinux.conf
+    generic-extlinux-compatible.enable = lib.mkDefault true;
+  };
+
+  # Enable additional firmware (such as Wi-Fi drivers).
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+
+  # Fix for not detecting the M.2 NVMe SSD. Will cause recompilation.
+  boot.kernelPackages = lib.mkDefault pkgs.linuxPackages_5_10;
```

Comment:
How about?

```suggestion
  boot.kernelPackages = lib.mkIf (lib.versionOlder pkgs.linux.version "5.10") lib.mkDefault pkgs.linuxPackages_latest;
```

Reply:
I don't think newer kernel would cause drivers to go away...

Reply:
Yeah, the config names MIGHT change, but it's unlikely. I'll do that.